package.json

```diff
@@ -107,5 +107,31 @@
     "stylelint-order": "^6.0.1",
     "vinyl": "^3.0.0",
     "wait-on": "^7.0.1"
+  },
+  "overrides": {
+    "chokidar@^2": {
+      "glob-parent": "^5"
+    },
+    "glob-stream@6": {
+      "glob-parent": "^5"
+    },
+    "latest-version@^5": {
+      "package-json": "^7"
+    },
+    "postcss-unmq": {
+      "postcss": "^7"
```

Comment:
PostCSS maintains plugin compatibility up to v7 (but not v8)

https://github.com/postcss/postcss/commit/54cbf3c4847eb0fb1501b9d2337465439e849734